pkg/apis/storage/types.go

```diff
@@ -65,8 +65,19 @@ type StorageClass struct {
 	// for all PVs created from this storageclass.
 	// +optional
 	AllowVolumeExpansion *bool
+
+	// VolumeBindingMode indicates how PersistentVolumeClaims should be
+	// provisioned and bound.  When unset, VolumeBindingImmediate is used.
+	// This field is alpha-level and is only honored by servers that enable
+	// the VolumeTopologyBinding feature.
+	// +optional
+	VolumeBindingMode *VolumeBindingMode
 }
 
+// NoProvisioner is a special Provisioner value to use when a StorageClass
+// is required but there is no dynamic provisioner.
+const NoProvisioner = "kubernetes.io/no-provisioner"
```

Comment:
Why do we need this, as opposed to ""

also, this seems unrelated to this PR?

Reply:
Validation requires that the provisioner string is non-empty.  Also, when we add dynamic provisioning support in the future, we will need a way to distinguish between a StorageClass that only supports static provisioning (with some fake provisioner name) vs dynamic provisioners.  But you're right, we don't need it right now.  I can add it in a future PR when it will actually get used.